src/core/providers/gdal/qgsgdalprovider.cpp

```diff
@@ -485,6 +486,29 @@ void QgsGdalProvider::reloadProviderData()
   ( void )initIfNeeded();
 }
 
+void QgsGdalProvider::loadMetadata()
+{
+  // Set default, may be overridden by stored metadata
+  mLayerMetadata.setCrs( crs() );
+
+  if ( mDriverName == QLatin1String( "OpenFileGDB" ) )
```

Comment:
If someone wanted to expand this beyond OpenFileGDB, are there other drivers worth exploring? This is a general question, not bound to this PR :)

Reply:
> If someone wanted to expand this beyond OpenFileGDB, are there other drivers worth exploring? This is a general question, not bound to this PR :)

Yes, if there's other GDAL formats which have metadata available to them which we can map across to QGIS metadata fields!